docs/docs/guides/getting_started.md

```diff
@@ -8,5 +8,108 @@ sidebar_label: Getting Started
 
 Welcome to Web3.js Documentation.
 
-web3.js is a collection of libraries that allow you to interact with a local or remote ethereum node using HTTP, IPC or WebSocket.
-The following documentation will guide you through different use cases of Web3.js, upgrading from older versions as well as providing an API reference documentation with examples.
+web3.js is a collection of libraries that allow you to interact with a local or remote Ethereum node using HTTP, IPC or WebSocket.
+The following documentation will guide you through different use cases of Web3.js, upgrading from older versions, as well as providing an API reference documentation with examples.
+
+### Some major features of Web3.js v4 are:
+
+-    Web3.js [Plugins Feature](/guides/web3_plugin_guide/) for extending functionality ( [List of Existing Plugins](https://web3js.org/plugins) )
+-    ESM and CJS builds 
+-    [Tree shakable with ESM](/guides/advanced/web3_tree_shaking_support_guide/)
+-    [Contracts dynamic types](/guides/smart_contracts/infer_contract_types_guide/) & full API in TypeScript
+-    Using native BigInt instead of large BigNumber libraries
+-    More efficient ABI Encoder & Decoder
+-    Custom Output formatters
+-    In compliance with Eth EL API
+
+### Installation
+
+If NPM is being used as package manager, use the following for installing the web3.js library. 
+
+```
+npm i web3
+```
+
+For installing using yarn package manager:
+
+```
+yarn add web3
+```
+
+Note: Installing web3.js in this way will bring in all web3.js sub-packages, if you only need specific packages, it is recommended to install the specific required packages. 
+
+### Importing and using Web3.js
+
+Web3.js v4 supports both CJS ( CommonJS ) and native ESM module imports. For importing the main Web3 class in CJS you can use:
+
+``` js
+const { Web3 } = require('web3');
+```
+
+and for ESM style imports, you can use:
+
+``` ts
+import { Web3 } from 'web3';
+
+```
+
+### Providers and Transport
+
+Web3.js is in compliance with [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193) so any EIP-1193 provider can be injected in web3.js . There are HTTP, WebSocket and IPC providers also available as web3.js packages for using.
+
+Following are examples of creating web3 instance with HTTP provider. 
+
+``` ts
+import { Web3, HttpProvider } from 'web3';
+const web3 = new Web3('https://mainnet.infura.io/v3/YOUR_PROJ_ID');
+web3.eth.getBlock('latest').then(console.log);
+// or
+const web3_2 = new Web3(new HttpProvider('https://mainnet.infura.io/v3/YOUR_PROJ_ID'));
```

Comment:
It will be better to use the web3 name here

Reply:
its 2nd instance of web3 obj in code example :) first is named web3